src/jdk.compiler/share/classes/com/sun/tools/javac/jvm/Gen.java

```diff
@@ -44,6 +44,9 @@
 import com.sun.tools.javac.tree.EndPosTable;
 import com.sun.tools.javac.tree.JCTree.*;
 
+import java.util.HashMap;
```

Comment:
sorry one final matter of style these imports should come first before all the `com.sun.tools.javac.*` ones, there is no need for another review iteration just please fix it before integrating. Thanks!

Reply:
Fixed.